GDJS/Runtime/events-tools/soundtools.ts

```diff
@@ -428,6 +428,24 @@ namespace gdjs {
         runtimeScene
           .getSoundManager()
           .unloadAudio(soundFile, /* isMusic= */ true);
+
+      export const interpolateAudioTo = (
+        runtimeScene: gdjs.RuntimeScene,
+        channel: integer,
+        toVolume: float,
+        timeOfFade: float
+      ) => {
+        const sound = runtimeScene.getSoundManager().getSoundOnChannel(channel);
+        if (sound)
+          runtimeScene
+            .getSoundManager()
+            .interpolateAudioTo(channel, toVolume, timeOfFade);
```

Comment:
```suggestion
          sound.fade(sound.getVolume(), toVolume, timeOfFade);
```

Move the code from howler-sound-manager.js here.